CONTRIBUTING.md

```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
+[code-of-conduct]: http://todogroup.org/opencodeofconduct/#react-server/opensource@redfin.com
```

Comment:
When I go to the code of conduct, the top of the page has this now:

![image](https://cloud.githubusercontent.com/assets/933552/13437180/b3f46e8a-df97-11e5-9938-7b94504ac5ee.png)